rag/infrastructure/celery/workers/pipeline_tasks.py

```
@@ -38,11 +38,16 @@ def build_context(source_type: str, source_data: dict) -> PipelineContext:
     # Extract (same as backend lines 25-26)
     pipeline_id = source_data.get("pipeline_id")
     metadata_dict = source_data.get("metadata")
-    
+        
     # Validate (same as backend lines 27-28)
     if not pipeline_id or not metadata_dict:
         raise ValueError("Pipeline ID or metadata not found in source_data")
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)